omnigan/trainer.py

```diff
@@ -101,6 +103,73 @@ def __init__(self, opts, comet_exp=None, verbose=0, device=None):
             self.exp = comet_exp
         self.domain_labels = {"s": 0, "r": 1}
 
+    @torch.no_grad()
+    def paint(self, image_batch, mask_batch=None, resolution="approx"):
```

Comment:
@tianyu-z @51N84D @alexrey88 @melisandeteng please have a look at this and tell me if this makes sense to y'all